src/events/http2.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * @fileoverview Handlers for http & https libraries instrumentation
+ */
+
+const uuid4 = require('uuid4');
+const shimmer = require('shimmer');
+const urlLib = require('url');
```

Comment:
is this a core library? is it available in all versions or should you tryRequire it as well?

Reply:
Yes, this is from version 4.x. It's also being used in the original http module.